Client/Frontend/UserContent/UserScripts/Scripts_Dynamic/Scripts/DomainSpecific/Paged/BraveTalkScript.js

```diff
@@ -20,4 +20,23 @@ window.__firefox__.includeOnce("BraveTalkScript", function($) {
       }
     }
   });
+  
+  const launchNativeBraveTalk = $(function (url) {
+    $.postNativeMessage('$<message_handler>', {
+      'kind': 'launchNativeBraveTalk',
+      'url': url,
+      'securityToken': SECURITY_TOKEN
+    });
+  });
+  
+  if (document.location.host === "talk.brave.com") {
+    const postRoom = (event) => {
```

Comment:
Must be `const postRoom = $((event => {...})`

Reply:
Fixed in https://github.com/brave/brave-ios/pull/6020/commits/cfb28e8c93e67492749e9b55143167528b34ffd6